modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-resizing.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
+
+You can enlarge a virtual machine (VM) disk by resizing the disk's persistent volume claim (PVC).
+
+VM disk enlargement makes extra space available to the virtual machine. However, it is the responsibility of the VM owner to decide how to consume the storage.
+
+Enabling VM disk enlargement notifies the virtual machine about size changes and, if the disk is a `Filesystem` PVC, the matching file expands to the remaining size while reserving some space for file system overhead.
+
+[NOTE]
+====
+You cannot reduce the size of a VM disk.
+====
+
+.Procedure
+
+. Edit a VM disk's PVC by running the following command:
++
+[source,terminal]
+
+----
+$ oc edit pvc <pvc_name>
+----
+
+. Increase the `spec.resource.requests.storage` item to a larger size in the manifest of the VM disk that you want to expand:
```

Comment:
In the PVC manifest, you are changing the value of an attribute. I'd suggest rephrasing this as:

````
Change the value of `spec.resource.requests.storage` attribute to a larger size.
````